init.js

```diff
@@ -51,12 +51,8 @@ require.cache[SHPath].exports = class WebSocketShard extends SH {
 	identify() {
 		let hotReload = this.manager.client.options.hotReload;
 		if(hotReload) {
-			const data = (hotReload.sessionData || this.manager.client._loadSession(this.id))?.[this.id]
-			if(data?.id && data.sequence > 0 && !this.sessionID && data.lastConnected + 60000 > Date.now()) {
-				this.sessionID = data.id;
-				this.closeSequence = this.sequence = data.sequence;
-			}
 			this.once(Constants.ShardEvents.RESUMED, () => {
```

Comment:
there should be a timeout to remove this event or some kind of check, perhaps also moving this to shardCreate() would be better.
If hotReload exists but the file for this id doesnt or expired, it will reidentify but this event will linger until a normal resume happens a few hours later.

Reply:
Very good point I didn't think of that. Perhaps moving it into `shardCreate()` is a better idea. Why did we have it in identify in the first place?